Podfile

```diff
@@ -8,10 +8,10 @@ workspace 'WordPress.xcworkspace'
 
 ## Pods shared between all the targets
 def shared_with_all_pods
-  pod 'CocoaLumberjack', '~> 3.2.0'
-  pod 'FormatterKit/TimeIntervalFormatter', '~> 1.8.1'
+  pod 'CocoaLumberjack', '3.2.0'
+  pod 'FormatterKit/TimeIntervalFormatter', '1.8.2'
   pod 'NSObject-SafeExpectations', '0.0.2'
-  pod 'UIDeviceIdentifier', '~> 0.1'
+  pod 'UIDeviceIdentifier', '~> 0.4'
```

Comment:
The newest version is 1.1.1, this is [missing](https://github.com/squarefrog/UIDeviceIdentifier/compare/0.5.0...1.1.1) several iPhone/iPad models. The API hasn't changed, so updating to the latest should be fine. Although I think we could just drop this one and use the raw platform string in logs.